ci-operator/step-registry/assisted/nutanix/postinstall/assisted-nutanix-postinstall-commands.sh

```diff
@@ -51,10 +51,209 @@ stringData:
 EOF
 echo "machine API credentials created"
 
-until \
-  oc wait --all=true clusteroperator --for='condition=Available=True' >/dev/null && \
-  oc wait --all=true clusteroperator --for='condition=Progressing=False' >/dev/null && \
-  oc wait --all=true clusteroperator --for='condition=Degraded=False' >/dev/null;  do
-    echo "$(date --rfc-3339=seconds) Clusteroperators not yet ready"
-    sleep 1s
+version=$(oc version | grep -oE 'Server Version: ([0-9]+\.[0-9]+)' | sed 's/Server Version: //')
+resource_timeout_seconds=300
+
+# Do the following if OCP version is >=4.13
+if [[ $(echo -e "4.13\n$version" | sort -V | tail -n 1) == "$version" ]]; then
+  echo "Found OCP version greater or equal to 4.13, creating cloud-provider-config ConfigMap"
+  cat <<EOF | oc apply -f -
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cloud-provider-config
+  namespace: openshift-config
+data:
+  config: "{\n\t\"prismCentral\": {\n\t\t\"address\": \"${NUTANIX_ENDPOINT}\",\n\t\t\"port\":${NUTANIX_PORT},\n\t\t\"credentialRef\": {\n\t\t\t\"kind\": \"Secret\",\n\t\t\t\"name\": \"nutanix-credentials\",\n\t\t\t\"namespace\": \"openshift-cloud-controller-manager\"\n\t\t}\n\t},\n\t\"topologyDiscovery\": {\n\t\t\"type\": \"Prism\",\n\t\t\"topologyCategories\": null\n\t},\n\t\"enableCustomLabeling\": true\n}\n"
```

Comment:
```suggestion
  config: |
    {
      "prismCentral": {
        "address": "${NUTANIX_ENDPOINT}",
        "port": ${NUTANIX_PORT},
        "credentialRef": {
          "kind": "Secret",
          "name": "nutanix-credentials",
          "namespace": "openshift-cloud-controller-manager"
        }
      },
      "topologyDiscovery": {
        "type": "Prism",
        "topologyCategories": null
      },
      "enableCustomLabeling": true
    }
```
can we make it a bit more readable?

Reply:
This is how it's saved in openshift and that didn't worked when I tried that. I'll try to insist more making it more readable